gulpfile.js

```diff
@@ -1,11 +1,38 @@
+'use strict';
+
+/*
+ * gulpfile.js
+ * ===========
+ * Rather than manage one giant configuration file responsible
+ * for creating multiple tasks, each task has been broken out into
+ * its own file in the 'gulp' folder. Any files in that directory get
+ *  automatically required below.
+ *
+ * To add a new task, simply add a new task file in that directory.
+ */
+
 var gulp = require('gulp');
-var conn = require('gulp-connect');
+var requireDir = require('require-dir');
+
+global.paths = {
+  // CSS sources
+  'css': './css/*',
```

Comment:
Although that's an excellent article, I disagree that, in this case, these comments are superfluous. My general rule for comments--aside from conforming to JavaDoc-style conventions on certain projects--looks something like this:

1. Could a novice programmer take more than three seconds to understand some code? If so, either comment, refactor, or both.
2. Is a comment turning into an explanation? If so, refactor. 

I can delete these comments if that's what you would prefer.